### PR TITLE
View

### PR DESCRIPTION
**Pull request by SimonGrchevski**
- Created routes and view
   - /users
   - /users/show
   - /users/:id/post
   - /users/:id/post/:id
- Add stylezzzzzz

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
   def index
-    @user = User.last_posts(current_user.name)
+    @user = User.all
   end
 
-  def show; end
+  def show
+    @user = User.find_by_id(params[:id])
+  end
 end
```
Comment (DeliceLydia) [inline app/controllers/users_controller.rb]: - [x] Remember you need to have a recent_post method that will display only 3recent posts of the user
and when a person wants to view all the user posts will hit `show all posts` link

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,20 @@
...
+  </section>
+  <section class="postsText my-3">
+    <p><%= @post.text%></p>
+  </section>
+
+  <section class="bord p-2 shadow">
+  <% @post.comments.each do |comment| %>
+    <p><%= comment.user.name %> : <%= comment.text %> 
+  <% end %>
+  </section>
```
Comment (DeliceLydia) [inline app/views/posts/show.html.erb]: - [x] When showing a post you should have only 5recent comments but currently you are rendering all comments please work on that

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @SimonGrchevski, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @SimonGrchevski 👋🏽
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**